llvm/include/llvm/MC/CAS/MCCASObjectV1.h

```diff
@@ -6,6 +6,99 @@
 //
 //===----------------------------------------------------------------------===//
 
+/// \verbatim
```

Comment:
I think this still needs to be attached to something. Can you move it into the main Class' doxygen comment?

Reply:
or you can make it into the `\file` comment.